web/src/components/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Helmet from 'react-helmet';
+// eslint-disable-next-line
+import typography from '../utils/typography';
+import logo from '../assets/logo.svg';
+import styles from './App.module.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className={styles.app}>
+        <Helmet>
+          <meta charSet="utf-8" />
+          <title>Realm-o-Matic</title>
+          <meta name="description" content="Self-service KeyCloak Realm Provosioner" />
+          <meta name="twitter:author" content="ShellyXueHan" />
+          <meta name="twitter:description" content="Self-service KeyCloak Realm Provosioner"
```

Comment:
yyou should have a logo for the realm-o-matic :) `twitter:image`

Reply:
will do when I have one :P